cloudformation/mongo-opsmanager-backup.template

```diff
@@ -16,6 +16,25 @@
         "PROD"
       ]
     },
+    "Stack": {
+      "Description": "Stack name",
+      "Type": "String"
+    },
+    "SshKeyFileS3Url": {
+      "Description": "S3 URL for a list of public SSH keys to use for the default ubuntu user",
+      "Type": "String",
+      "Default": ""
+    },
+    "EBSOptions": {
+      "Description": "Extra parameters to add-encrypted script",
+      "Type": "String",
+      "Default": "-t io1 -i 1000"
```

Comment:
Noticed you've changed the default here form 200 to 1000. Probably fine (200 is tiny for this), but thought I'd highlight it.